### PR TITLE
error checking for missing or incorrectly checking the blank checkbox…

### PR DESCRIPTION
**Pull request by Canada101**
… works and a message is displayed to the user identifying the issue. This brings about another issue that needs solving and that is once the program hits any of the defined error checking and fails it does not automatically reset

Fixes #issue_number

**What was changed?**
I added error checking for the no blanks checkbox either being clicked incorrectly or missed and having blanks
*Here, describe what part of the application you changed (e.g. login page, database, etc.). If possible, mention what specific components were added, removed, or modified.*
I changed the server logic to include checking the input file for blanks
**Why was it changed?**
This code was added to account for user error when interacting with the program
*Here, describe the issue that you are fixing with this code, and why your code fixes it.*
This code corrects the user error when incorrectly filling out the input fields and notifies them of what they need to change
**How was it changed?**
How it was specifically changed is there was no error checking on the button before and that lead to crashed with no description. This change allows the user to identify the error in input and as of now refresh the program and have the input be correct
*Here, get into detail about what files you modified, and talk about the most important lines in regards to fixing the issue.*
I modified the server file and the most important line is  "has_blanks <- any(is.na(dataset)) || any(dataset == "NA") " which scans the file for blanks the that value is checked against the checkbox allowing for error identification 

**How was it tested**
I tested this by running the same file in the described issue and getting the notification to pop up correctly when running the program 
*Here, get into detail about how you tested the files you modified.
I tested them by doing clicking and not clicking the checkbox for different files after seeing their correct inputs to determine the error was being caught successfully 
**SIDE NOTE**
One of the pull requests currently up from last week that is not merged is the reset button with reset functionality. If that code is merged I can incorporate the logic into the next sprint and have errors that are caught reset the fields so the user does not need to restart the program. 
**Screenshots that show the changes (if applicable):**

### CODE REVIEW

**Comment by avazzano7** [comment]
After reset button is created (this is in its own PR), this can be worked on further - fields need to be passed properly to avoid R crashing due to improper parameters

**Review by avazzano7 — CHANGES_REQUESTED** [review]
After reset button is created (this is in its own PR), this can be worked on further - fields need to be passed properly to avoid R crashing due to improper parameters

File: code/server.r
Comment (BollepalliPranitha) [inline code/server.r:1]: Even with the error notification, if users delay clicking "Understood" or even if they do click it, the app still crashes. This suggests that while the error notification logic is functional, additional changes may be necessary to stabilize the application during error states.
image
**Special Note:**
Extending the validation to scan for missing sequences before further processing can help avoid crashes and provide a clear message to users about missing required data.

**Review by BollepalliPranitha — CHANGES_REQUESTED** [review]

File: code/server.r
Comment (BollepalliPranitha) [reply code/server.r:1]: Try to mention Fixes #issue_number while creating the PR

**Comment by BollepalliPranitha** [comment]
This issue is being handled by another PR
https://github.com/oss-slu/MeltShiny/pull/221
So closing this issue

**Pull request closed without merging**